src/sass/public/UI/equipments/_equipments.scss

```
@@ -0,0 +1,169 @@
+@use 'base/variables' as v;
+@use 'base/mixins' as m;
+
+.equipments {
+    text-align: center;
+
+    &__heading {
+        margin-top: 2rem;
+        @include m.heading;
+
+        @include m.tablet{
+            font-size: 6rem;
+        }
+    }
+
+    @include m.desktop{
+        &__video {
+            width: 100%;
+        }
+    }
+
+    &__content {
+        position: relative;
+        overflow: hidden;
+        margin: -1rem 0 4rem 0;
+        @include m.phoneXL{
+            margin: 0 0 7rem 0;
+        }
+    }
+
+    &__grid {
+        position: absolute;
+        height: 100%;
+        width: 100%;
+        top: 0;
+        left: 0;
+        padding: 3rem;
+        @include m.grid(1, 1.5rem);
+        @include m.phoneXL{
+            @include m.grid(2, 4rem 2rem);
+            padding: 7rem 4rem;
+        }
+        @include m.tablet{
+            padding: 6rem 5rem;
+            row-gap: 3rem;
+        }
+        @include m.desktop {
+            padding: 4rem 5rem;
+            @include m.grid(3, 4rem 2rem);
+        }
+    }
+}
+
+
+
+.equipment {
+    border: 2px solid v.$white;
+    padding: .75rem;
+    height: 12rem;
+
+    &--center {
+        grid-column: 2 / 3;
+    }
+    
+    &__icon {
+        height: 6rem;
+        margin: 0 auto;
+
+        @include m.phoneXL{
+            height: 10rem;   
+            width: 12rem;
+        }
+
+        @include m.tablet{
+            height: 11rem;   
+        }
+
+        @include m.desktop{
+            height: 60%;
+            width: 35%;
+        }
+
+        @include m.desktopXL{
+            height: 60%;
+            width: 35%;
+        }
+    }
+
+    @include m.phoneXL{
+        padding: 2rem 2rem 0 2rem;
+        height: 21rem;
+    }
+
+    @include m.tablet{
+        padding: 1.5rem 2rem 2rem 2rem;
+        height: 23rem;
+    }
+
+    @include m.desktop{
+        padding-top: 2rem;
+        padding-bottom: 0;
+        height: auto;
+    }
+
+    &__name,
+    &__text {
+        color: v.$white;
+    }
+
+    &__name {
+        margin-bottom: 0;
+        font-size: 1.5rem;
+        font-weight: 500;
+
+        @include m.phoneXL{
+            font-size: 2.5rem;
+        }
+
+        @include m.desktop{
+            font-size: 2rem;
+        }
+
+        @include m.desktopXL{
+            font-size: 2.5rem;
+        }
+    }
+
+    &__text {
+        transition: opacity 300ms ease-in-out;
+        margin: -1rem 0 0 0;
+        font-size: 1.2rem;
+        font-weight: 200;
+
+        @include m.phoneXL{
+            font-size: 1.6rem;
+        }
+
+        @include m.desktop{
+            font-size: 1.4rem;
+        }
+
+        @include m.desktopXL{
+            font-size: 1.8rem;
+        }
+    }
+
+    &__icon,
+    &__name {
+        transition: transform 300ms ease-in-out;
+    }
+
+    transition: backdrop-filter 200ms ease-in-out;
+
+    &:hover {
+        backdrop-filter: blur(50px);
```

```suggestion
        backdrop-filter: blur(50px) hue-rotate(15deg) saturate(1.8);
```

add this to match the hover effect in the designs